src/core/server/saved_objects/import/import_saved_objects.ts

```diff
@@ -61,6 +62,7 @@ export async function importSavedObjectsFromStream({
   dataSourceTitle,
   workspaces,
   dataSourceEnabled,
+  assignedDataSources,
```

Comment:
```suggestion
  assignedDataSourcesInTargetWorkspace,
```
I feel it is less confusing than previous naming.

Reply:
updated